main.tf

```diff
@@ -93,10 +98,6 @@ resource "google_compute_instance_template" "atlantis" {
     enable_vtpm                 = true
   }
 
-  confidential_instance_config {
-    enable_confidential_compute = var.enable_confidential_compute
```

Comment:
The COS stable image doesn't support confidential compute, therefor it's removed.